forms-flow-api/src/formsflow_api/services/employeeDataService.py

```diff
@@ -0,0 +1,21 @@
+import json
+import requests
+from http import HTTPStatus
+
+class EmployeeDataService: 
+    
+    @staticmethod
+    def get_employee_data_from_bcgov(guid):
+      #TODO: get auth token from bcgov using @param guid
+      try:
+        responseFromBCGov = requests.get('https://analytics-testapi.psa.gov.bc.ca/apiserver/api.rsc/Datamart_Telework_employee_demo/',
+                       headers={'Authorization': 'Basic XXXXXXXXXXXXXXXXXXXX'})
```

Comment:
Any idea how to best add the token for testing? env?

@matbusby-fw, for context, currently we don't know how to authenticate to Telus endpoint so for new use a test token which I masked with `X` in this pr.

Reply:
@iman-jamali-fw I would suggest that we move this URL + the auth token to the config file (`config.py`) so it can be different for each env.

Reply:
+ would have to modify the docker-compose file to pass it along